runtime/runtime-core/jvm/test/aws/smithy/kotlin/runtime/retries/impl/StandardRetryIntegrationTestResources.kt

```diff
@@ -184,5 +187,38 @@ val standardRetryIntegrationTestCases = mapOf(
                 expected:
                   outcome: max_attempts_exceeded
                   retry_quota: 480
-        """.trimMargin(),
+        """.trimIndent(),
+)
+
+/* ktlint-disable annotation spacing-between-declarations-with-annotations */
```

Comment:
Should this rule be permanently disabled? I noticed we run into this a lot

Reply:
I still favor this rule the vast majority of the time. Kotlin is a denser language than Java with the capability to encode much more meaning in shorter lines but that can quickly hamper readability and maintainability. Rules like this keep us from attempting to be obscuredly brief unless we have a really good reason.

In my mind, those good reasons include many similar declarations requiring the same annotation which are otherwise lightly differentiated and have no user-facing documentation. These serialization classes are a good example of that because they only exist to hold data in the format already specified by the JSON struct...they carry no business logic, aren't user-facing, etc. The only other place I see we've disabled this rule is for the [typed environment setting implementations](https://github.com/awslabs/smithy-kotlin/blob/b4580e3d4a6fbf81023ba6379e1704a86b6b1c1b/runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/config/EnvironmentSetting.kt#L53-L58) which again are not user-facing and are largely rote (i.e., they differ only by type).

In the end, I like this rule and would strongly prefer keeping it—even at the expense of removing these overrides and putting the attributes on separate lines for these simple cases.